### ISSUE TITLE
P2 compatibility?

### ISSUE THREAD

**Issue by mrferrar** [Brewskey/spark-server#302]
Hello, 

I've been doing some testing to see if a P2 will work with your server, and am running into some issues. 
When trying to force the protocol to tcp during the 'particle keys' commands like you suggest for 'Electron and newer hardware', the protocol change doesn't seem to take affect. The P2 still says it's using UDP, and I'm worried it may be locked to that. And the server doesn't seem to recognize the P2 trying to connect to it via UDP.

Have you done any successful P2 testing yourself, or have any plans to look into it in the future?

Thanks

**Comment by jlkalberer** [comment]
Yeah, the server doesn't support UDP. I haven't ever tried to make it work
but it might be easy. You could just try to set up a UDP socket server
alongside the TCP socket server.
